bookkeeper-server/src/main/java/org/apache/bookkeeper/client/api/ReadHandle.java

```diff
@@ -115,6 +115,20 @@
      */
     long getLength();
 
+    /**
+     * Returns whether the ledger is sealed or not.
+     *
+     * <p>A ledger is sealed when either the client explicitly closes it ({@link WriteHandle#close()} or
+     * {@link WriteAdvHandle#close()}) or another client explicitly open and recovery it
+     * {@link OpenBuilder#withRecovery(boolean)}.
+     *
+     * <p>This method only checks the metadata cached locally. The metadata can be not update-to-date because
+     * the metadata notification is delayed.
+     *
+     * @return true if the ledger is sealed, otherwise false.
+     */
+    boolean isSealed();
```

Comment:
If we don't rename the operation that seals the ledger, we shouldn't leave this as isClosed().